.vscode/settings.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Please remove personal IDE config files.

Reply:
Deleted the file in another commit. Unsure if rebasing is needed or not.